net/dnscrypt-proxy/files/dnscrypt-proxy.init

```diff
@@ -33,7 +33,7 @@ create_config_file() {
     config_get      client_key      $1 'client_key'     ''
     config_get      syslog_prefix   $1 'syslog_prefix'  'dnscrypt-proxy'
     config_get      query_log_file  $1 'query_log_file' ''
-    config_get      log_level       $1 'log_level'	''
+    config_get      log_level       $1 'log_level'	'5'
```

Comment:
I see the confusion and why it's different on LEDE/OpenWRT.  In that second link you posted, log level 6 would cause more output than log level 5.  However, in LEDE/OpenWRT, the log level settings work the opposite (for dnscrypt).  Log level 6 causes less output than log level 5.  The default log level of dnscrypt-proxy is LOG_INFO.  Which would correspond to log level 5 in LEDE/OpenWRT.  

There are two log level settings in system configuration in openwrt/lede: conloglevel and cronloglevel.  Conloglevel works the way the second link says: higher numbers give more info.  Cronloglevel works the opposite: higher numbers give less info.  

My only guess is that dnscrypt in openwrt/lede follows the way Cronloglevels work.  The higher the number, the less information displayed.

Test out my additions yourself.  You'll see at level 5 (which is the default in this case) info messages (and more severe) are displayed from dnscrypt in the system log.  At level 6, only notice messages (and more severe) are logged.